src/components/Budget/BudgetView/BudgetView.js

```diff
@@ -3,73 +3,81 @@ import PropTypes from 'prop-types';
-      <Row end="xs">
-        <Col xs={12}>
-          <ExpandAllButton
-            accordionStatus={sections}
-            onToggle={expandAll}
-          />
-        </Col>
-      </Row>
-      <AccordionSet
-        accordionStatus={sections}
-        onToggle={toggleSection}
-      >
-        <Accordion
-          label={<FormattedMessage id="ui-finance.budget.summary.title" />}
-          id={SECTIONS_BUDGET.SUMMARY}
-        >
-          {budget.metadata && <ViewMetaData metadata={budget.metadata} />}
-          <BudgetSummary
-            budget={budget}
-            fiscalYearCurrency={fiscalYearCurrency}
-          />
-        </Accordion>
-        <Accordion
-          label={<FormattedMessage id="ui-finance.budget.information.title" />}
-          id={SECTIONS_BUDGET.INFORMATION}
-        >
-          <BudgetInformation
-            allowableEncumbrance={budget.allowableEncumbrance}
-            allowableExpenditure={budget.allowableExpenditure}
-            awaitingPayment={budget.awaitingPayment}
-            budgetStatus={budget.budgetStatus}
-            encumbered={budget.encumbered}
-            expenditures={budget.expenditures}
-            fiscalEnd={fiscalEnd}
-            fiscalStart={fiscalStart}
-            fiscalYearCurrency={fiscalYearCurrency}
-            name={budget.name}
-            id={budget.id}
-            overEncumbrance={budget.overEncumbrance}
-            overExpended={budget.overExpended}
-          />
-        </Accordion>
-      </AccordionSet>
+      <AccordionStatus>
+        <Row end="xs">
+          <Col xs={12}>
+            <ExpandAllButton />
+          </Col>
+        </Row>
+        <AccordionSet>
+          <Accordion
+            label={<FormattedMessage id="ui-finance.budget.summary.title" />}
+            id={SECTIONS_BUDGET.SUMMARY}
+          >
+            {budget.metadata && <ViewMetaData metadata={budget.metadata} />}
+            <BudgetSummary
+              budget={budget}
+              fiscalYearCurrency={fiscalYearCurrency}
+            />
+          </Accordion>
+          <Accordion
+            label={<FormattedMessage id="ui-finance.budget.information.title" />}
+            id={SECTIONS_BUDGET.INFORMATION}
+          >
+            <BudgetInformation
+              allowableEncumbrance={budget.allowableEncumbrance}
+              allowableExpenditure={budget.allowableExpenditure}
+              awaitingPayment={budget.awaitingPayment}
+              budgetStatus={budget.budgetStatus}
+              encumbered={budget.encumbered}
+              expenditures={budget.expenditures}
+              fiscalEnd={fiscalEnd}
+              fiscalStart={fiscalStart}
+              fiscalYearCurrency={fiscalYearCurrency}
+              name={budget.name}
+              id={budget.id}
+              overEncumbrance={budget.overEncumbrance}
+              overExpended={budget.overExpended}
+            />
+          </Accordion>
+
+          {Boolean(expenseClassesTotals.length) && (
```

Comment:
looking at proptypes `expenseClassesTotals` is not required, so I expect it could be undefined

Reply:
should be `required`